de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/eventhandler/NegotiationHandler.java

```diff
@@ -17,11 +17,12 @@
 import de.fu_berlin.inf.dpp.intellij.ui.wizards.JoinSessionWizard;
 import de.fu_berlin.inf.dpp.monitoring.IProgressMonitor;
 import de.fu_berlin.inf.dpp.negotiation.FileList;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'de.fu_berlin.inf.dpp.negotiation.FileList'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWc2FtjV0EcSutaxNIIh&open=AWc2FtjV0EcSutaxNIIh)

[ISSUE_KEY]: AWc2FtjV0EcSutaxNIIh